samples/Basic_rBoot/app/application.cpp

```diff
@@ -163,8 +217,21 @@ void init() {
 	Serial.begin(SERIAL_BAUD_RATE); // 115200 by default
 	Serial.systemDebugOutput(true); // Debug output to serial
 	
+	uint8 slot = 0;
+
+#ifdef BOOT_RTC_ENABLED
+	debugf("Temp Mode is enabled");
```

Comment:
rtc being enabled doesn't necessarily mean you are in temp boot mode
also, currently running rom is printed further down this method already...